funding/manager.go

```diff
@@ -462,6 +464,22 @@ type Config struct {
 	// MaxAnchorsCommitFeeRate is the max commitment fee rate we'll use as
 	// the initiator for channels of the anchor type.
 	MaxAnchorsCommitFeeRate chainfee.SatPerKWeight
+
+	// RequestAlias allows the Manager to request a unique ShortChannelID
+	// alias. This is only used for option_scid_alias channels.
```

Comment:
I think we should break these out into a distinct interface. It'll crowd this config struct less, and also make it easier to mock things out or even expose with a layer of indirection over the sub-servers.

Reply:
done